test/hotspot/gtest/runtime/test_committed_virtualmemory.cpp

```diff
@@ -159,7 +159,7 @@ class CommittedVirtualMemoryTest {
     const size_t page_sz = os::vm_page_size();
     const size_t num_pages = 4;
     const size_t size = num_pages * page_sz;
-    char* base = os::reserve_memory(size, !ExecMem, mtTest);
+    char* base = os::reserve_memory(size, mtTest, !ExecMem);
```

Comment:
```suggestion
    char* base = os::reserve_memory(size, mtTest);
```

Reply:
ExecMem is a parameter, that could be false.